R/importTaxpasta.R

```diff
@@ -0,0 +1,116 @@
+#' Import taxpasta-specific BIOM results to `TreeSummarizedExperiment`
+#'
+#' @param filename A character vector with one element that denotes the file
+#'   path to a BIOM file.
+#'
+#' @return A [TreeSummarizedExperiment::TreeSummarizedExperiment()] object.
+#' @export
+#'
+#' @examples
+#' tse <- importTaxpasta("profiles.biom")
```

Comment:
When the package is built, manual pages are also run. These example does not work since there are no "profiles.biom" file. You can disable the run bu \dontrun (or something like that)

Reply:
or, if the file is inserted in one of the inst/ subfolders then it can be called in examples with the system.file function (see other R functions that use this)

Reply:
That would be even better solution.

Data goes to inst/extdata and python script goes to inst/scripts